qiskit_research/utils/dynamical_decoupling.py

```diff
@@ -95,7 +95,7 @@ def get_instruction_durations(backend: Backend) -> InstructionDurations:
 
 # TODO refactor this as a CalibrationBuilder transpilation pass
 def add_pulse_calibrations(
-    circuits: Union[QuantumCircuit, list[QuantumCircuit]],
+    circuits: Union[QuantumCircuit, List[QuantumCircuit], List[List[QuantumCircuit]]],
```

Comment:
Same comment as above, would prefer not to allow list of lists because it deviates from what's in Qiskit.

Reply:
Removed the `List[List[QuantumCircuit]]`.